data/config.yaml

```diff
@@ -12,7 +12,7 @@ data_update_lookback_seconds: 3600
 timeout: [30.0, 30.0]
 
 # sets the logging level. Possible values are: [CRITICAL, ERROR, WARNING, INFO, DEBUG]
-logging_level: INFO
+logging_level: DEBUG
```

Comment:
Not sure if DEBUG should be the default

Reply:
Good catch. That is temporary change, which I forgot to undo. Fixing.